american-journal-of-enology-and-viticulture.csl

```diff
@@ -0,0 +1,207 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
```

Comment:
@POBrien333, can you check the locale? The guide says American-English, but I don't know if "en-US" would e.g. mess up the quotes.

Reply:
Thanks. Checked and indeed, en-US is better a fit.